assets/section-main-product.css

```diff
@@ -1473,3 +1473,129 @@ a.product__text {
     display: none;
   }
 }
+
+/* Recipient form */
+.recipient-form {
+  display: block;
+  position: relative;
+  --recipient-checkbox-line-height: 2.88rem;
+  /* (line-height - 1.6rem) / 2 */
+  --recipient-checkbox-margin-top: 0.64rem;
```

Comment:
I previously gave the opposite feedback to Fred when he wrote this 😂.

My thinking is that a calc using nothing but constants gives no benefit for dynamism as you have to go and manually change each constant each time you want to make a change _and_ it's worse for performance because you're spending clock cycles doing a calculation that will always be the same.

This answers your first question, no that variable is not used anywhere it's left over from when this was a calc, I have removed it and updated the comment.

I haven't put it back to a calc for reasons outlined above but if you prefer that, we can do it. Thanks.